cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/schedule/ProgramScheduleMeta.java

```diff
@@ -24,20 +24,18 @@
 public class ProgramScheduleMeta {
 
   private final ProgramScheduleStatus status;
-  private final long lastUpdated;
+  private final long lastUpdateTime;
 
   public ProgramScheduleMeta(ProgramScheduleStatus status, long lastUpdated) {
     this.status = status;
-    this.lastUpdated = lastUpdated;
+    this.lastUpdateTime = lastUpdated;
   }
 
   public ProgramScheduleStatus getStatus() {
     return status;
   }
 
-  public long getLastUpdated() {
-    return lastUpdated;
-  }
+  public long getLastUpdateTime() { return lastUpdateTime; }
```

Comment:
style: code blocks in {} should be on a new line.

Reply:
fixed.